gateway-shell/pom.xml

```diff
@@ -31,6 +31,10 @@
     <description>Client DSL shell.</description>
 
     <dependencies>
+    	<dependency>
+  			<groupId>org.apache.commons</groupId>
+  			<artifactId>commons-math3</artifactId>
+		</dependency>
```

Comment:
Indentation is incorrect.

Reply:
Corrected.

Reply:
I still see it wrong, but it should not prevent merging this change.

Reply:
Odd, it appeared fine for me in my IDE both times, I'll try to play around with it some more, not sure why it is doing that.

Reply:
tabs vs spaces?

Reply:
That looks like what it was, I've replaced all whitespace there with spaces in the newest patch so hopefully that is all it was.